src/main/java/frc/robot/subsystems/swerve/module/stuffs/DriveStuff.java

```diff
@@ -0,0 +1,17 @@
+package frc.robot.subsystems.swerve.module.stuffs;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.motor.ControllableMotor;
+import frc.robot.hardware.request.IRequest;
+import frc.robot.hardware.signal.InputSignal;
+
+public record DriveStuff(
```

Comment:
consider changing the name to something like `DriveAvatiach` or `DriveProperties`

Reply:
"AVATIACH"